nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/NXRuntimeTestCase.java

```diff
@@ -469,14 +469,20 @@ public void deployContrib(String uri) throws Exception {
      *
      * @param bundle the bundle that becomes the contribution owner
      * @param contrib the contribution to deploy as part of the given bundle
+     * @Deprecated since 10.1 with @LocalDeploy
```

Comment:
Javadoc is `@deprecated`, not `@Deprecated`.
For the message follow the pattern `@deprecated since 10.1, use {@code #deployContrib} instead`